docs/dev.md

```diff
@@ -147,6 +147,8 @@ You need to install Podman or the Docker runtime (Docker Desktop on macOS or Win
 
 Your project must have a Containerfile or Dockerfile to use dev mode in container mode. An example of container configuration is shown in [Building an application image](https://github.com/openliberty/ci.docker/#building-an-application-image). The parent image must be one of the [Open Liberty container images](https://github.com/openliberty/ci.docker/#container-images), or an image using Linux with Open Liberty configured with the same paths as the Open Liberty container images. The container config file must copy the application .war file and the server configuration files that the application requires into the container.
 
+In case you are using a custom image, you need to add `--user 1001` (or whatever user id the image should run with) to your `containerRunOpts` configuration variable. You cal also set it on the command line using `"-DcontainerRunOpts=--user 1001"`.
```

Comment:
Thank you for the example and update to the doc Ben. You have one misspelling in the second sentence. "You cal" should be "You can"

Reply:
I fixed it for you.